data/ambientlight.json

```diff
@@ -1,6 +1,31 @@
 {
   "TR": "https://www.w3.org/TR/ambient-light/",
   "impl": {
-    "caniuse": "ambient-light"
+    "caniuse": "ambient-light",
+    "edgestatus": "Ambient Light Sensor API",
+    "chromestatus": 5298357018820608,
+    "other": [
+      {
+        "ua": "firefox",
+        "status": "",
+        "source": "feedback",
+        "date": "2018-03-16",
```

Comment:
Should it be 2018-01-03 (the date #160 was created)?

Reply:
Actually, I believe this should rather be the date when someone last reviewed and validated the information to ease maintenance over time, as in "Hmm, that info hasn't been updated in two years, is it still valid? Yes, ok, let's update the date to assert that it's still current".

We could distinguish between a "creation date" and a "last reviewed date" but that seems overkill. If needed later on, the creation date can be retrieved from Git history.

Reply:
Sounds fine to me.